src/test/function_test/config.ini

```diff
@@ -51,6 +51,7 @@ random_seed = 0
 [network]
 ; how many network threads for network library(used by asio)
 io_service_worker_count = 4
+primary_interface = @LOCAL_HOSTNAME@
```

Comment:
`primary_interface` is considered as a network interface name, e.g. `eth0`, `lo`, why assign it a hostname?